src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerT.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Extensions.Logging
     [DebuggerDisplay("{DebuggerToString(),nq}")]
     public class Logger<T> : ILogger<T>
     {
+        [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
```

Comment:
Out of curiosity, is there any way we can make the debugger not display all private fields without needing to annotate each individual field?